### PR TITLE
Tidy some CLI stuff.

### PR DESCRIPTION
**Pull request by pepyakin**
Some insubstantial clean up of CLI output performed by Gav on rococo-branch

### CODE REVIEW

**Review by expenses — APPROVED** [review]

**Comment by pepyakin** [comment]
bot merge

File: bin/node/cli/src/browser.rs [hunk 1]
```diff
@@ -48,7 +48,7 @@ async fn start_inner(chain_spec: Option<String>, log_level: String) -> Result<Cl
 	info!("✌️  version {}", config.impl_version);
 	info!("❤️  by Parity Technologies, 2017-2020");
 	info!("📋 Chain specification: {}", config.chain_spec.name());
-	info!("🏷  Node name: {}", config.network.node_name);
+	info!("🏷 Node name: {}", config.network.node_name);
```
[the remark this answers is not in the document]
Comment (jasl) [reply bin/node/cli/src/browser.rs:51]: the extra space shouldn't be removed

image

[the remark this answers is not in the document]
Comment (jasl) [reply bin/node/cli/src/browser.rs:51]: macOS's default Terminal looks wrong too

image

What terminal do you use?

[the remark this answers is not in the document]
Comment (bkchr) [reply bin/node/cli/src/browser.rs:51]: Konsole from KDE.

https://stackoverflow.com/questions/7086856/programmatically-tell-if-a-unicode-character-takes-up-more-than-one-character-sp/7088274

not sure there is something we can do.

[the remark this answers is not in the document]
Comment (jasl) [reply bin/node/cli/src/browser.rs:51]: I see.

I was found other Emoji can lead this too (parachains if my memory correct),
although it is not a big deal, but it looks uncomfortable for OCD.

if possible, could you change another emoji for this?

[the remark this answers is not in the document]
Comment (bkchr) [reply bin/node/cli/src/browser.rs:51]: Which would you recommend?

I also don't find this a nice solution... Perfect would be if we could find out in advance if we need to add another whitespace..

[the remark this answers is not in the document]
Comment (jasl) [reply bin/node/cli/src/browser.rs:51]: I also don't have better idea 🤦

**Pull request merged**